src/api/utils.js

```diff
@@ -156,6 +157,10 @@ export const parseUserSaveData = (
         addValueAsProp(cred, values[propName], propName)
     )
 
+    if (!cred[EXPIRE_DATE] && typeof cred[EXPIRE_DATE] !== 'undefined') {
+        delete cred[EXPIRE_DATE]
+    }
```

Comment:
It's very good you caught this. Please add a comment above line 160 with a link to the backend bugfix issue.

Reply:
Added to the commit